controls/linux-headers_exists.rb

```diff
@@ -0,0 +1,13 @@
+plan_name = input("plan_name", value: "core/linux-headers")
```

Comment:
Looks good, couple of suggestions (along with the caveat that I haven't tested the below ;) 

```
# Might be clearer to gather inputs at the top of the file
plan_files = input("plan_files")
plan_name = input("plan_name", value: "core/linux-headers")
plan_dir = input("plan_dir", value: "include")

hab_pkg_path = command("hab pkg path #{plan_name}")
# could optionally check that the package path command was successful
describe hab_pkg_path do
  its('exit_status') { should eq 0 }
  its('stdout') { should_not be_empty }
end

# File.join insulates from users setting trailing slashes or not
target_dir = File.join(hab_pkg_path.stdout.strip, plan_dir)

# As we're using the file resource let's loop over plan_file instead of file for readability
plan_files.each do | plan_file |  
  describe file(File.join(target_dir, plan_file)) do
    # These file resource checks might be useful
    it { should exist }
    it { should be_file }
    its('size') { should > 0 }
  end
end
```

Reply:
I'll give these a test, thanks for the suggestions.